explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -263,20 +261,11 @@ object ObsTabTiles:
             .whenA(itcProps.isExecutable)
             .runAsyncAndForget
       }
-      // ITC selected target. Here to be shared by the ITC tile body and title
```

Comment:
This is an example of a state that was pushed down to the tile where it makes more sense